src/components/VHeader/VHeader.vue

```diff
@@ -1,22 +1,64 @@
 <template>
-  <div
-    class="sticky top-0 flex py-4 px-6 md:px-7 align-center justify-between z-40 w-full bg-white"
+  <header
+    class="sticky top-0 flex py-4 px-4 md:px-7 items -center justify-between z-40 w-full bg-white gap-x-2"
     :class="{
+      'border-b border-white': !isHeaderScrolled && !isFilterSidebarVisible,
       'border-b border-dark-charcoal-20':
-        isHeaderScrolled || isFilterSidebarVisible,
+        isSearchRoute && (isHeaderScrolled || isFilterSidebarVisible),
+      'flex-wrap gap-y-4': !isMinScreenMD && !isHeaderScrolled,
     }"
   >
-    <NuxtLink to="/">
+    <NuxtLink
+      to="/"
+      class="rounded-sm ring-offset-1 focus:outline-none focus-visible:ring focus-visible:ring-pink -ms-2 inline-flex items-center hover:bg-yellow"
+      :class="{
+        'pe-3': !isHeaderScrolled || !isSearchRoute,
+        'md:px-0': isSearchRoute,
+      }"
+    >
       <VLogoLoader :status="isFetching ? 'loading' : 'idle'" />
+      <svg
+        v-if="!isHeaderScrolled || !isSearchRoute"
+        class="-ml-1 mt-1"
```

Comment:
This is what is causing the text still appear, I think the second part can be ommited.
```suggestion
      <svg
        v-if="!isHeaderScrolled"
        class="ml-1 mt-1"
```